Makefile

```diff
@@ -116,7 +116,7 @@ manifests: controller-gen ## Generate manifests e.g. CRD, RBAC etc.
 
 # Run e2e tests locally using go build while also setting up a proxy in the shell to allow
 # the test to run as if it were inside the cluster. This enables mongodb connectivity while running locally.
-e2e-telepresence: cleanup-e2e install ## Run e2e tests locally using go build while also setting up a proxy
+e2e-telepresence: cleanup-e2e install ## Run e2e tests locally using go build while also setting up a proxy e.g. make e2e-telepresence test=replica_set cleanup=true
 	telepresence connect; \
 	telepresence status; \
```

Comment:
If we're running this as you suggest (e.g. `make e2e-telepresence test=replica_set cleanup=true`), we probably don't need `telepresence status` here. It might be worth considering to remove it.

Reply:
It does make sense. 
But TBH, the command was already there and I only added a comment how it could be used. If you think it makes sense, I can remove it. 🤔

Reply:
I have not used telepresence a lot to give enough feedback, but I thought keeping the status might be nice to have a small overview before the test starts? 
At least thats what I think the initial author wanted? 
But I have no strong opinion, if you think removing it is better, I can do so 👍

Reply:
Yeah, let's remove it.

Reply:
Done: https://github.com/mongodb/mongodb-kubernetes-operator/pull/1184/commits/c10566af8be713899efbcef8716fba28b0113c1c!